README.md

```diff
@@ -17,23 +17,34 @@ projects but the idea is to share what we've made.
 $ ./gradlew assemble
 ```
 
-* Publish to bintray:
+* Publish to Jitpack (for `Tracks` >`1.2.1`)
 
-```sh
-$ ./gradlew assemble publishToMavenLocal bintrayUpload -PbintrayUser=XXX -PbintrayKey=XXX -PdryRun=false
-```
-
-Note: running the `publishToMavenLocal` task is very important, it will
-create the .po file needed for any maven repository.
+There's no need to do anything. Jitpack will handle build of requested verison itself.
 
 ## Usage
 
+### Jitpack
```

Comment:
It might be worth adding a link to Jitpack build instructions in this section in case our README gets out of sync. (not necessarily to this line)

Reply:
That's a good idea. I've ended up with removing custom instruction I wrote and adding link to Jitpack instruction as duplicating this indeed seems redundant. Does it work for you?